handlers/create.go

```diff
@@ -31,6 +31,11 @@ func CreateMachine(event *events.Event, apiClient *client.RancherClient) error {
 		return handleByIdError(err, event, apiClient)
 	}
 
+	// If name is not supplied, error out
```

Comment:
While you're in there, if you could figure out how validChars/invalidChars works and disallows spaces, that would be an awesome bonus.